octavia_f5/restclient/as3restclient.py

```diff
@@ -93,7 +123,10 @@ def reauthorize(self):
         LOG.debug("Reauthorized!")
 
     @authorized
+    @_metric_post_exceptions.count_exceptions()
```

Comment:
Could we move the count_exceptions decorator above the authorized decorator since post/patch/delete will probably trigger the exceptions counts for every reauthorize, but we catch that count already seperatly?

Reply:
Ah sorry, I though the reauthorization result code causes a exception, but meanwhile changed it so it doesn't raise for http errors. We probably should catch the result codes seperately